src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -135,7 +135,7 @@
 	public String aspectRatioVideoTrack;
 	private int videoBitDepth = 8;
 
-	private byte thumb[];
+	private volatile byte[] thumb = null;
```

Comment:
Why should multiple threads write to this variable? Is thumbnail regenerated in different size for each thread/request?